docs/ops/normalization/MVN_6.md

```diff
@@ -0,0 +1,91 @@
+## MVN <a name="MVN"></a> {#openvino_docs_ops_normalization_MVN_6}
+
+**Versioned name**: *MVN-6*
+
+**Category**: *Normalization*
+
+**Short description**: Calculates mean-variance normalization of the input tensor.
+
+**Detailed description**
+
+*MVN* subtracts mean value from the input blob:
+\f[
+o_{i} = i_{i} - \frac{\sum{i_{k}}}{C * H * W}
+\f]
+If *normalize_variance* is set to 1, the output blob is divided by variance:
+\f[
+o_{i}=\frac{o_{i}}{\sum \sqrt {o_{k}^2+\epsilon}+\epsilon}
+\f]
+
+**Attributes**
+
+* *normalize_variance*
+
+  * **Description**: *normalize_variance* is a flag that specifies whether to perform variance normalization.
+  * **Range of values**:
+    * `false` -- do not normalize variance
+    * `true` -- normalize variance
+  * **Type**: `boolean`
+  * **Default value**: `false`
+  * **Required**: *no*
+
+* *eps*
+
+  * **Description**: *eps* is the number to be added to the variance inside sqrt to avoid division by zero when normalizing the value. For example, *epsilon* equal to 0.001 means that 0.001 is added to the variance.
```

Comment:
`*eps* is the number to be added to the variance inside sqrt...` - is outdated. Please, make description more generic.

Reply:
And remove sentence "For example, *epsilon* equal to 0.001 means that 0.001 is added to the variance." which looks really redundant.